core/client/app/utils/link-component.js

```diff
@@ -6,8 +6,8 @@ LinkComponent.reopen({
     active: computed('attrs.params', '_routing.currentState', function () {
```

Comment:
This ones an exception, I think we should stick as closely as possible to the core CP signature: https://github.com/emberjs/ember.js/blob/v2.4.4/packages/ember-routing-views/lib/components/link-to.js#L585